src/components/PostList/PostList.jsx

```diff
@@ -1,3 +1,23 @@
-export const PostList = () => (
-  <>Put the list here</>
+import './PostList.scss';
+import { PostInfo } from '../PostInfo';
+
+export const PostList = ({ posts, commentsList, users }) => (
+  <div className="PostList">
+    {posts.map((post) => {
+      const postExtra = {
+        ...post,
+        user: users.find(user => post.userId === user.id),
+        comments: commentsList
+          .filter(comment => comment.postId === post.id),
+      };
```

Comment:
I think it is better to prepare the data before App even runs, so before the first render